tests/base/model_template.py

```diff
@@ -66,7 +66,7 @@ def __init__(
 
         # if you specify an example input, the summary will show input/output for each layer
         # TODO: to be fixed in #1773
```

Comment:
is this fixed? @awaelchli

Reply:
I was just gonna ask about this. Will there be an issue if I uncomment it? Coz the PR seems to be in progress.

Reply:
yeah you can un-comment the example input array and remove this todo comment. 
maybe someone commented it because the input size does not match, let me know if it causes issues. 

>  Coz the PR seems to be in progress.

not it is merged.